test/integration/api/admin/api_spec.js

```diff
@@ -921,6 +921,67 @@ describe("api", function () {
       });
     });
   });
+  describe('delete_backed_up_assets', function () {
+    it('should delete specific version IDs', async function () {
+      this.timeout(TIMEOUT.LARGE);
+
+      // Process:
+      // - Upload to the same public ID three times
+      // - Delete a single version (string)
+      // - Delete a two versions (array)
+      // - Cleanup
+
+      // Perform three uploads
+      const firstUpload = await uploadImage({
+        public_id: PUBLIC_ID_BACKUP_1,
+        backup: true
+      });
+      await wait(1000)();
```

Comment:
Once you `await` an upload, if it returns, it means that the request finished. There's no need for an additional `wait`, it only makes the tests run much longer than needed.

Reply:
That's what I figured, but there are multiple `await wait(1000)();`s in the same file, so I thought it was a convention I didn't know about :)

Removed!